scripts/root2hdf5.py

```diff
@@ -103,7 +103,8 @@ def root2hdf5(filename1,filename2, output):
 	image["event_index"] = -1
 	image.append()
 	k=1
-	for i in range(0,230):
+	total_events = len(charge1)
+	for i in range(0,total_events):
 		if eventid1[i] != 0:
 			i2 = np.where(eventid2==eventid1[i])
 			i2 = int(i2[0])
```

Comment:
This method doesn't create the map we expect. We want the event ids to be same for both cams irrespective of their location in the eventid arrays. While this method maps all nonzero ids at same positions in the array (For example, it will map StereoEvtNumber 18 from M1 to 14 on M2 because they're both at position 20 in respective arrays).

Reply:
Ah I got it now with the help of your slides from the call! It's a pity to search for every StereoEvtNumber in the other array. The np.where should be fast, but maybe we can optimise here. We only need to perform the search on the coming array slices, since the values are ascending.